src/gtfs/mod.rs

```diff
@@ -551,3 +600,24 @@ pub fn write_to_zip<P: AsRef<std::path::Path>>(model: Model, path: P) -> Result<
     input_tmp_dir.close()?;
     Ok(())
 }
+
+/// gtfs pickup or drop off method "Must coordinate with driver to arrange pickup" (type 3)
+/// corresponds to pickup or drop off method "upon reservation only" in ntfs (type 2)
+pub fn convert_gtfs_pickup_drop_off_type_to_ntfs_type(collections: &mut Collections) {
```

Comment:
What a long name. How about something a bit shorter like `convert_pickup_dropoff_to_ntfs` (since we already are in `gtfs` module anyway so no need to keep it and the word `type` is probably not that important?).

Reply:
At first I had implemented it in model and I wanted an explicit name. But you’re right it’s no longer necessary